README.md

```diff
@@ -59,7 +60,7 @@ $ curl http://localhost:9080/v1/GetRateLimits \
 
 ### ProtoBuf Structure
 
-An example rate limit request sent via GRPC might look like the following
```

Comment:
I'm not sure if it's strictly necessary but it _may_ be worth adding a note that gRPC was dropped? Just for anyone who may have been using Gub with gRPC has an explicit warning